### PR TITLE
<CalendarPanelFooter /> - create a new story page

### PR DESCRIPTION
**Pull request by bamai**
### 🔦 Summary
https://jira.wixpress.com/browse/DSM-596
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/CalendarPanelFooter/docs/examples.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const standard = `<CalendarPanelFooter/>`;
```
Comment (EgoziE) [inline src/CalendarPanelFooter/docs/examples.js:1]:
```suggestion
export const standard = `
<CalendarPanelFooter
  dateToString={date => date.toLocaleDateString('he-IL')}
  primaryActionLabel="Apply"
  secondaryActionLabel="Cancel"
  selectedDays={{
    from: new Date('2019-05-27T21:00:00.000Z'),
    to: new Date('2019-05-27T21:00:00.000Z'),
  }}
/>
`;
```

File: src/CalendarPanelFooter/docs/index.story.js [hunk 2]
```diff
@@ -23,4 +44,42 @@ export default {
     primaryActionLabel: 'submit',
     secondaryActionLabel: 'cancel',
   },
+
+  sections: [
+    header({
+      sourceUrl: `https://github.com/wix/wix-style-react/tree/master/src/${CalendarPanelFooter.displayName}/`,
+      component: <CalendarPanelFooter />,
```
Comment (EgoziE) [inline src/CalendarPanelFooter/docs/index.story.js:51]: Can be removed :)

File: src/CalendarPanelFooter/docs/index.story.js [hunk 3]
```diff
@@ -23,4 +44,42 @@ export default {
...
+    }),
+
+    tabs([
+      tab({
+        title: 'Description',
+        sections: [
+          description({
+            title: 'Description',
+            text: 'A component that permits the user to make a binary choice',
```
Comment (EgoziE) [inline src/CalendarPanelFooter/docs/index.story.js:60]: A footer for the CalendarPanel component

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5525/

**Pull request merged**